Updates for apply page - job guarantee landing page - financing page - new CTA component

ArielCalisaya opened the pull request:
## Linked issues

#### From Aug, 29, 2025
- https://github.com/breatheco-de/breatheco-de/issues/9663
- https://github.com/breatheco-de/breatheco-de/issues/9664
- https://github.com/breatheco-de/breatheco-de/issues/9671

#### From Sept. 2, 2025
- https://github.com/breatheco-de/breatheco-de/issues/9658
- https://github.com/breatheco-de/breatheco-de/issues/9662

#### From Sept. 5, 2025 - Merging Financing page and new CTA component:
- https://github.com/breatheco-de/breatheco-de/issues/9656
- https://github.com/breatheco-de/breatheco-de/issues/9738
- https://github.com/breatheco-de/breatheco-de/issues/9349
- https://github.com/breatheco-de/breatheco-de/issues/9728

#### From Sept 10, 2025
- https://github.com/breatheco-de/breatheco-de/issues/9711
   - https://github.com/breatheco-de/breatheco-de/issues/9729 (Duplicated)

#### From Sept 12, 2025
- https://github.com/breatheco-de/breatheco-de/issues/9754

#### From Sept 15, 2025
- https://github.com/breatheco-de/breatheco-de/issues/9808

#### From Sept 16, 2025
- https://github.com/breatheco-de/breatheco-de/issues/9764

#### From Sept 18, 2025
- https://github.com/breatheco-de/breatheco-de/issues/9831
  - https://github.com/breatheco-de/breatheco-de/issues/9796 (Duplicated)

#### From Sept 23, 2025
- https://github.com/breatheco-de/breatheco-de/issues/9863

#### From Sept 24, 2025
- https://github.com/breatheco-de/breatheco-de/issues/9879
- https://github.com/breatheco-de/breatheco-de/issues/9888

#### From Sept 25, 2025
- https://github.com/breatheco-de/breatheco-de/issues/9903
- https://github.com/breatheco-de/breatheco-de/issues/9908

#### From Sept 29, 2025
- https://github.com/breatheco-de/breatheco-de/issues/9918
- https://github.com/breatheco-de/breatheco-de/issues/9917

#### From Sept 30, 2025
- https://github.com/breatheco-de/breatheco-de/issues/9928

OmarHG098 committed:
UPDATE: components in financial page

OmarHG098 committed:
ADD: new paymment plans component

OmarHG098 committed:
Merge branch 'development' into revamp/finance-page

OmarHG098 committed:
UPDATE: payment component

OmarHG098 committed:
UPDATE: change component order

OmarHG098 committed:
ADD: template rule

OmarHG098 committed:
Merge branch 'breatheco-de:master' into revamp/finance-page

OmarHG098 committed:
UPDATE: fiancial page components

ArielCalisaya committed:
feat: Add SafeReCAPTCHA wrapper to prevent crashes without GATSBY_CAPTCHA_KEY

ArielCalisaya committed:
(SafeReCAPTCHA): use forwardRef import directly from react

ArielCalisaya committed:
updated landing "Job Guarantee"

ArielCalisaya committed:
update content of apply page

ArielCalisaya committed:
visual_fix(apply.us): update paragraph styling in header section

ArielCalisaya committed:
docs: fix punctuation in job guarantee text

ArielCalisaya committed:
UI fixes: update payment UI and content for better clarity

- Improve toggle accessibility by wrapping in label
- Update payment plan titles and descriptions
- Optimize job guarantee info rendering
- Adjust grid layout for better responsiveness
- Add text-wrap property to Heading component

ArielCalisaya committed:
style(PricesAndPayment): update background colors and layout spacing

ArielCalisaya committed:
Add React import in SafeReCAPTCHA component for consistency. Define image field types for course instructors in GraphQL schema to support instructor images.

ArielCalisaya committed:
Use direct React.forwardRef instead of importing forwardRef separately

ArielCalisaya committed:
Merge remote-tracking branch 'upstream/master' into development

OmarHG098 committed:
Merge branch 'revamp/finance-page' of https://github.com/OmarHG098/website-v2 into revamp/finance-page

ArielCalisaya committed:
Improve right section content with clearer bullet points in spanish version

ArielCalisaya committed:
Merge remote-tracking branch 'upstream/development' into development

OmarHG098 committed:
UPDATE: finance page

ArielCalisaya committed:
Add: Explaining Gatsby markdown processing behavior

This comment clarifies that any .md file in /src/**/*.md will be processed
as a page by Gatsby, which helps developers understand why README.md files
in component folders can cause build issues.

ArielCalisaya committed:
style: format SafeReCAPTCHA component and its usage

ArielCalisaya committed:
Merge remote-tracking branch 'omar-changes/revamp/finance-page' into omar/financing-pages

ArielCalisaya committed:
Merge branch 'omar/financing-pages' into update/financing-page

ArielCalisaya committed:
feat(WeTrust): add full background option for we_trust section

Add bg_full field to control background styling of the we_trust section. When enabled, the background color will span the full width instead of being contained within the content area.

ArielCalisaya committed:
docs(landing): update job guarantee page content and structure

Update badges section and adjust positions for rating reviews.
Modify heading text for who is hiring section to be more impactful.

ArielCalisaya committed:
feat(WeTrust): add label support and improve responsive styling

- Add label field to graphql query and yml data structure
- Improve tablet layout with auto height and flex styling
- Enhance title display with label support and better typography

ArielCalisaya committed:
feat(WeTrust): add bg_full field and improve box label styling

- Add bg_full field to we_trust_section in template and data files
- Update label styling in WeTrust component for better consistency
- Fix text formatting in job guarantee data files

ArielCalisaya committed:
style(FaqCard): adjust width to 45rem for better responsiveness

ArielCalisaya committed:
feat(payment-plans): enhance payment options UI and content structure

- Restructure payment plans content with detailed loan options and CTAs
- Implement nested bullet points and improved formatting in payment plans component
- Update both English and Spanish payment plans content for consistency

ArielCalisaya committed:
style: formatting and spacing in css and js files

ArielCalisaya committed:
Add background wrapper and full-width support

implement BackgroundWrapper component to handle full-width backgrounds
update TwoColumn and PaymentPlans components to use new wrapper
add bg_full prop to enable full-width background sections

ArielCalisaya committed:
improve code readability by removing unnecessary line breaks

ArielCalisaya committed:
feat: add double action CTA component with career consultation and newsletter signup

- Create new DoubleActionCTA component with two interactive cards
- Add PageWrapper to handle CTA display logic
- Include YAML content files for US and ES languages
- Update styling and form components to support new features
- Modify Layout to fetch and pass CTA data

ArielCalisaya committed:
feat(DoubleActionCTA): improve styling and update success messages

- Add --font-sans CSS variable for consistent typography
- Update success messages in both English and Spanish locales
- Make calendar icon stroke color customizable via props
- Adjust padding and typography styles for better visual hierarchy
- Fix newsletter form message references and loading state colors

ArielCalisaya committed:
feat(WeTrust): add label support and improve component styling

- Add label field to boxes in graphql queries and yml files
- Implement label display in WeTrust component with new styling
- Improve responsive layout and background handling
- Adjust box shadow and dimensions for better tablet display

ArielCalisaya committed:
code formatting and readability across components

ArielCalisaya committed:
feat: improve button responsiveness in home template

ArielCalisaya committed:
styleformatting

ArielCalisaya committed:
fix: change paragraph to paragraph_html in Header component

Update the Header component to use paragraph_html prop instead of paragraph to properly handle HTML content. This ensures consistent rendering of formatted text.

ArielCalisaya committed:
style(components): improve readability of Header component props formatting

ArielCalisaya committed:
Add DataTable component

- Create reusable DataTable component for displaying data tables
- Add responsive design with sticky headers
- Implement CSS grid system (2-5 columns recommended)
- Add hover effects for primary and secondary buttons
- Support for HTML content and custom styling
- Optimized scrollbar styles for better UX

ArielCalisaya committed:
style: clean up code formatting and fix indentation

- Remove extra whitespace in program-comparison.js
- Standardize hex color case in style.css
- Simplify SubTitle component logic in Sections/index.js
- Fix YAML description length in program-comparison.es.yml
- Clean up DataTable component formatting and comments

ArielCalisaya committed:
fix(Sections): Unify conditional rendering pattern with dangerouslySetInnerHTML

- Add verify-node-version.js for Node.js version compatibility checks
- Update package.json scripts to include version verification
- Fix Header component to simplify conditional rendering logic

ArielCalisaya committed:
style(verify-node-version): standardize string quotes and improve error messages

- Replace single quotes with double quotes for string consistency
- Enhance error messages with more detailed explanations and solutions
- Improve formatting of version check output for better readability

ArielCalisaya committed:
feat: node version verification script with dynamic command message

Add support for passing custom command names to verify-node-version.js
Update package.json scripts to pass descriptive command names

ArielCalisaya committed:
style: standardize string quotes in verify-node-version.js

ArielCalisaya committed:
feat: update payment plans and styling across components

- Add position prop to Paragraph styled component
- Update payment plan titles and descriptions in US and ES locales
- Adjust WeTrust component styling with position and spacing
- Fix typo in financial.us.yml and update job guarantee section
- Modify prices_and_payment titles and select text in both locales
- Update financial.js template with new titleProps and query

ArielCalisaya committed:
Merge branch 'page/program-comparision' into merge/financing-page_and_program-comparison

ArielCalisaya committed:
feat(DataTable): enhance table component with icon support and styling

- Add CellContent component to handle icon rendering and text alignment
- Improve table styling with better padding and border radius handling
- Update component to use data_table instead of comparison_table
- Add support for icon properties in cells including position and color
- Refactor button rendering to use Link component consistently

ArielCalisaya committed:
style: fix indentation and formatting in financial and DataTable components

ArielCalisaya committed:
feat(financial): add contact form component and translations

Add new ContactForm component with validation, error handling and accessibility features
Include translations in financial.us.yml for contact form text and messages

ArielCalisaya committed:
fix(financial): handle undefined data_table in financial template

Prevent potential runtime errors by safely accessing nested data_table properties with optional chaining and providing a default empty object

ArielCalisaya committed:
Merge remote-tracking branch 'upstream/development' into development

ArielCalisaya committed:
Merge branch 'merge/financing-page_and_program-comparison' into development

ArielCalisaya committed:
feat(DynamicForm): replace ContactForm with dynamic configurable form component

Add new DynamicForm component with internationalization support and dynamic field configuration
Remove deprecated ContactForm component
Update financial page template to use new DynamicForm
Add translations for DynamicForm in English and Spanish

ArielCalisaya committed:
style: format code

ArielCalisaya committed:
docs(program-comparison): update currency format from USD to $ symbol

ArielCalisaya committed:
fix(verify-node-version): swap nvm install and switch version instructions

The order of quick solutions was incorrect. Installing should come before switching versions.

ArielCalisaya committed:
feat(ChooseYourProgram): add conditional rendering and container styles

Add hideEmptyContent prop to conditionally render empty content sections
Pass through containerStyles prop to customize container styling

ArielCalisaya committed:
feat(program-comparison): add choose program section to comparison page

Add new choose_program section to both English and Spanish versions of the program comparison page. Include corresponding component integration and styling adjustments.

ArielCalisaya committed:
feat(DataTable): add alternating row colors and clean up commented code

Implement row pattern styling for better readability by adding alternating background colors. Remove unused commented code for alignment properties.

ArielCalisaya committed:
refactor(DataTable): extract table components and constants into separate files

Move table-related components (CellContent, ActionButtons, TableHeader, TableCell, TableRow) and styling constants to dedicated files for better maintainability and organization

ArielCalisaya committed:
style(DataTable): standardize string quotes and improve code formatting

ArielCalisaya committed:
docs(DataTable): improve JSDoc comments with type definitions

Add detailed type definitions for DataAction, DataCellObject, DataCell and DataColumn.
Update component documentation to be more concise while maintaining all key information.

ArielCalisaya committed:
docs(DataTable): improve comments in constants file

ArielCalisaya committed:
style(DataTable): clean up formatting in constants file

ArielCalisaya committed:
feat(DataTable): add button variants and improve styling consistency

- Add button variant support (default, outline, link) to DataTable actions
- Extract button styles to constants for better maintainability
- Update CSS hover states to match new button variants
- Add variant fields to program comparison YAML files

ArielCalisaya committed:
style(DataTable): fix formatting and indentation in constants and components

ArielCalisaya committed:
fix(actions): add error handling to form submission functions

Wrap form submission logic in try-catch blocks to handle errors gracefully
Return consistent error response format when exceptions occur

ArielCalisaya committed:
formatting in actions.js

ArielCalisaya committed:
refactor: remove DynamicForm component and related files

The DynamicForm component and its associated translation files were removed as they are no longer needed in the financial template. This cleanup helps maintain a more focused codebase by removing unused code.

ArielCalisaya committed:
feat(Header): add color variant and text wrap support

Add colorConfig to support different color variants and implement textWrap prop

ArielCalisaya committed:
feat(application): add application process page with stepper component

Add new application process page template with English and Spanish content files.
Implement Stepper component to display application steps with hover effects and responsive design.

ArielCalisaya committed:
docs(DoubleActionCTA): update comment from Spanish to English

ArielCalisaya committed:
feat(application-process): enhance stepper hover effects and add apply button

- Extract box shadow logic in Stepper component for better readability
- Add apply button with hover and active states in application process
- Include iconogram section with bootcamp statistics
- Improve button animations with cubic-bezier timing function

ArielCalisaya committed:
style(Button): improve readability by breaking long lines in Button component

Refactor long lines into multiple lines for better readability and maintainability

ArielCalisaya committed:
docs: fix formatting in job guarantee stats

ArielCalisaya committed:
feat(success-stories): update content and add rating reviews section

- Update SEO titles, headers and descriptions for both English and Spanish versions
- Add new rating_reviews section with badges and star ratings
- Implement responsive layout for the new rating component

ArielCalisaya committed:
update partner section titles in success stories

ArielCalisaya committed:
Add success stories title and description to testimonials components in both English and Spanish versions. Also include a new testimonial entry in the US version.

ArielCalisaya committed:
feat(success-stories): enhance success stories page with featured testimonials and CTA

- Add featured testimonial section with video support
- Implement new CTA section for application conversion
- Update testimonial data structure and styling
- Improve button hover and active states
- Localize content for both English and Spanish

ArielCalisaya committed:
code formatting and readability

ArielCalisaya committed:
fix(Testimonials): filter testimonials with student_thumb

Ensure testimonials displayed have both valid visibility and student thumbnail

ArielCalisaya committed:
refactor(testimonials): improve testimonial filtering and image handling

- Simplify testimonial filtering logic in Testimonials component
- Extract student thumb and thumbnail video variables for clarity in SuccessStories
- Remove commented out student image code in FeaturedTestimonialSection

ArielCalisaya committed:
feat(application-process): add image to header

ArielCalisaya committed:
style: remove unnecessary parentheses around svg_image prop

ArielCalisaya committed:
feat(testimonials): add james testimonial and improve layout

fix testimonial splitting logic when no featured testimonial exists
adjust button width and positioning in success stories template
add decorative vectors to application process header

ArielCalisaya committed:
code formatting in success stories components

ArielCalisaya committed:
Merge remote-tracking branch 'upstream/master'

ArielCalisaya committed:
Merge branch 'master' into dev-2

alesanchezr committed:
Merge branch 'development' into development

ArielCalisaya committed:
feat(financial): add custom CTA to financial pages and update translations

- Add hideGlobalCTA flag to financial page meta_info
- Implement custom DoubleActionCTA component for financial pages
- Update CTA text in prices_and_payment components
- Refactor DoubleActionCTA to support new props and layout options

ArielCalisaya committed:
code readability with consistent formatting

ArielCalisaya committed:
Merge remote-tracking branch 'upstream/development' into development

ArielCalisaya committed:
Merge branch 'development' of github.com:ArielCalisaya/website-v2 into development

ArielCalisaya committed:
docs(testimonials): update student name and slug in us testimonials

ArielCalisaya committed:
Merge 'upstream/development' into development and fix conflicts

ArielCalisaya committed:
formatting and trailing commas

ArielCalisaya committed:
fix: correct student name spelling and update testimonial content

Update Hernan García to Hernán García in both Spanish and English testimonials
Improve English testimonial content for better readability and flow

ArielCalisaya committed:
refactor(cta): simplify financial page cta components and content

ArielCalisaya committed:
style(DoubleActionCTA): improve code formatting and readability

ArielCalisaya committed:
feat(prework): add prework page with content and styling

Add English and Spanish content files for prework page
Implement responsive layout with sections for what you'll learn, how it works, and why prework matters
Include styled components and interactive elements for better user experience

ArielCalisaya committed:
style: format prework.js with consistent quotes and spacing

ArielCalisaya committed:
Merge remote-tracking branch 'upstream/development' into development

OmarHG098 commented on src/components/DataTable/constants.js:
Tener las constantes aquí es un poco inconsitente con el proyecto, debería venir de /Styling para poder ser re-utilizable
Aunque si no lo vamos a usar en otros lados tampoco veo mayor motivo para que vengan desde ahí

ArielCalisaya committed:
clean(DoubleActionCTA): remove unused lang prop

ArielCalisaya committed:
feat(application-process): add DoubleActionCTA and hide global CTA

The page now uses a custom DoubleActionCTA component instead of the global CTA. Added hideGlobalCTA flag to meta_info in both English and Spanish versions.

ArielCalisaya committed:
refactor(DataTable): remove constants.js and consolidate styling

ArielCalisaya committed:
refactor: simplify table-components constants

- Remove unused BORDER_RADIUS, PADDING, and TYPOGRAPHY objects
- Replace with simple constants: BUTTON_BORDER_RADIUS, BUTTON_PADDING, SMALL_FONT_SIZE
- Keep only values that are actually used in the components
- Improve code maintainability and reduce complexity

ArielCalisaya committed:
style(table-components): improve code formatting and readability

ArielCalisaya committed:
improve UI of CTA and background color

OmarHG098 committed:
add description to choose program component

ArielCalisaya committed:
Merge pull request #1 from OmarHG098/patch-1

ArielCalisaya committed:
Merge remote-tracking branch 'upstream/development' into development

ArielCalisaya committed:
Merge branch 'development' into development

ArielCalisaya committed:
Merge remote-tracking branch 'origin/development' into development

OmarHG098 committed:
REFACTOR: double cta component uses static query, REFACTOR: location filtering based on existing locations

OmarHG098 committed:
UPDATE: group 6617 image

OmarHG098 committed:
ADD: application process to index page

ArielCalisaya committed:
Merge pull request #2 from OmarHG098/updates/ariel-development

Updates/ariel development

ArielCalisaya committed:
Merge remote-tracking branch 'upstream/development' into development

ArielCalisaya committed:
code formating

ArielCalisaya committed:
update awards page

ArielCalisaya committed:
code format

ArielCalisaya committed:
feat(prework): refactor components and update content structure

- Add boxTitleProps to WeTrust component for better title styling
- Restructure why_prework_matters section to use boxes instead of bullets
- Update SimpleCards component with new props and improved styling
- Enhance TwoColumn component with style parsing utility and container props
- Simplify prework template using reusable components

ArielCalisaya committed:
style(components):  code formatting and consistency

- Add trailing commas to props in component declarations
- Fix indentation and alignment in JSX elements
- Standardize prop object formatting across components

ArielCalisaya committed:
feat(navbar): add program comparison link in navbar and htmlTitle support

- Add program comparison CTA to navbar dropdown menus
- Include id field for navbar items
- Support htmlTitle in Header component for formatted titles
- Update awards page layout and styling

ArielCalisaya committed:
style:  code formatting and indentation in multiple components

- Add missing comma in Awards component containerStyle
- Fix spacing in Header component children prop
- Improve MegaMenu component formatting in both desktop and mobile versions

ArielCalisaya committed:
feat(partners): update partners page content

- Add new partnership section with image and call-to-action
- Update partner, coding, and financial sections content
- Improve page layout and remove redundant elements
- Add new Bite-Con partner logo
- Update form section with enhanced value proposition

ArielCalisaya committed:
style(partners): indentation and trailing whitespace in partners.js

ArielCalisaya committed:
feat(partners): update partner content and UI components

- Update program titles in Spanish localization
- Add center alignment for job guarantee title
- Modify partner financials section with new featured images
- Replace TwoColumn with ScholarshipProjects component
- Adjust marquee speed and featured images logic

ArielCalisaya committed:
refactor(marquee): simplify marquee component and improve performance

- Replace complex calculations with simpler CSS animations
- Remove unused imports and redundant code
- Add support for configurable duration and gap
- Improve image rendering consistency

alesanchezr committed:
Update index.js

alesanchezr reviewed (commented):
La landing de job-guarantee no carga:
https://website-v2-26cceewaj-geeksacademy.vercel.app/us/landing/job-guarantee


react-dom.production.min.js:189 ReferenceError: landingHiriging is not defined
    at Object.who_is_hiring (index.js:949:18)
    at landing_a.js:173:34
    at Array.map (<anonymous>)
    at landing_a.js:171:10
du	@	react-dom.production.min.js:189
index.js:949 Uncaught (in promise) ReferenceError: landingHiriging is not defined
    at Object.who_is_hiring (index.js:949:18)
    at landing_a.js:173:34
    at Array.map (<anonymous>)
    at landing_a.js:171:10

ArielCalisaya committed:
feat(prework): update prework page content and styling

- Add location filtering for schedule display
- Update icon colors and types in why_prework_matters section
- Remove note section and update CTA background color
- Make icon colors customizable via props
- Adjust schedule title formatting and add timezone

ArielCalisaya committed:
Merge remote-tracking branch 'origin/development' into development

ArielCalisaya committed:
Merge and fix conflicts

ArielCalisaya committed:
feat(Landing): add hiring data variable and fix job guarantee formatting

Add landingHiriging variable to access who_is_hiring data
Fix job guarantee section formatting by removing space in months range

ArielCalisaya committed:
style(partners): adjust spacing and background in partners template

Move background color to parent container and update margins/paddings for better visual consistency

ArielCalisaya committed:
docs(navbar): update partner link title in navbar us yaml

OmarHG098 committed:
ADD: tracking to cta button on succes-stories

ArielCalisaya committed:
docs(footer): update program names and add cybersecurity in footer

Update program names to include AI references and add new cybersecurity program to both English and Spanish footer sections

ArielCalisaya committed:
feat(Partners): update styling and content structure

- Capitalize text in PartnersCarousel title
- Update calendar icon to calendar-2 in partner yaml files
- Adjust font sizes in ScholarshipProjects
- Reorder components and hide images in partners template
- Increase font size for H3 in partners template

ArielCalisaya committed:
fix(WorkTogether): adjust spacing and visibility based on showImages prop

Make margin and padding values dynamic depending on showImages flag
Add conditional rendering for divider and adjust icon colors

ArielCalisaya committed:
style(prework): update styling and spacing for better visual consistency

- Change background colors and padding values
- Adjust card styling with border and alignment
- Simplify section spacing and update typography

ArielCalisaya committed:
style(prework): adjust padding and margins for better spacing

Update padding values to use consistent rem units and remove horizontal padding where unnecessary. Adjust margins to use rem units for better scalability. Also center-align button content by adding justifyContent and width properties.

ArielCalisaya committed:
comment out unused video container section

ArielCalisaya committed:
feat(partners): update partner images and testimonials text

- Replace gray-scale partner images with colored versions
- Fix capitalization and punctuation in Spanish testimonials
- Update company name references to use consistent casing

OmarHG098 committed:
UPDATE: Refactor DoubleActionCTA component
- Add staff assignment algorithms (ROUND_ROBIN and FIRST)
- Add priority field to staff members
- Remove locations from CTA query
- Add assignment_algo to YAML config
- Improve staff selection logic

OmarHG098 committed:
REMOVE: Global DoubleActionCTA injection
- Remove PageWrapper.js completely
- Remove DoubleActionCTA query from Layout.js
- Simplify Layout.js children rendering

OmarHG098 committed:
UPDATE: Add DoubleActionCTA to course templates
- Add DoubleActionCTA to course templates
- Pass flattened props from YAML
- Add action tracking to application process button

OmarHG098 committed:
REMOVE: hideGlobalCTA from page YAML files
- Remove hideGlobalCTA from application-process
- Remove hideGlobalCTA from financial
- Remove hideGlobalCTA from prework

OmarHG098 committed:
UPDATE: Data Science course name
- Update to 'Data Science and Machine Learning with AI'
- Update in choose_your_program
- Update in navbar
- Update in orlando-scholarship

OmarHG098 committed:
UPDATE: Template DoubleActionCTA integration
- Add DoubleActionCTA to remaining course templates
- Pass flattened props in financial.js
- Pass flattened props in index.js
- Remove hideGlobalCTA from GraphQL queries

ArielCalisaya committed:
docs: update prework page header images

ArielCalisaya committed:
Merge pull request #3 from OmarHG098/ariel-development

Refactorización de componente DoubleActionCTA

ArielCalisaya committed:
fix(prework): update page titles and button texts for better readability

- Split header titles into multiple lines for better display
- Update button texts to be more action-oriented
- Adjust layout and styling for responsive design

ArielCalisaya committed:
style: adjust spacing and layout in components

increase title font size and margin in ScholarshipProjects
update margin and remove maxHeight in PartnersCarousel
improve formatting in prework template

ArielCalisaya committed:
Merge remote-tracking branch 'origin/development' into development

ArielCalisaya committed:
feat(staff): update staff job titles and add new team member

- Update job titles for several staff members to reflect current roles
- Add new team member Laura as Program Manager USA
- Remove commented out staff entries for Tomas and Gustavo
- Keep consistent changes across both English and Spanish versions

ArielCalisaya committed:
fix: update Spanish URLs in prework page to correct paths

ArielCalisaya committed:
feat(the-academy): update content and layout for academy pages

- Add new partner logo image
- Update staff member names and details in yaml files
- Implement two-column layout component
- Revise academy page content with new mission-focused messaging
- Add partner images section and styling
- Adjust paragraph styles and spacing in components

ArielCalisaya committed:
style: replace single quotes with double quotes and remove console log

Clean up code formatting by standardizing string quotes and removing debug statement

ArielCalisaya committed:
chore: remove Macarena Rebolledo from staff data files

ArielCalisaya committed:
docs: update credential statistics in US and ES locales

Update alumni count from 6000 to 7000 and campuses from 7 to 8 in both US and Spanish locale files

ArielCalisaya committed:
feat(the-academy): update content and styling for academy pages

- Add fontFamily_title prop to Why component header
- Update icon colors and sizes in cornerstones section
- Modify campus count and student numbers in content
- Remove redundant partner list text
- Update icon names and add color properties

ArielCalisaya committed:
feat(Badges): add bg_full prop to make background transparent

Add new prop to optionally make Badges component background transparent while keeping other styling

ArielCalisaya committed:
style(Badges): standardize quote usage in background prop

ArielCalisaya committed:
feat(content): add links to prework pages in multiple course and landing pages

Update various YAML files to include hyperlinks to prework pages in English and Spanish versions. Also modify Stepper and ProgramDetails components to support HTML in text fields.

ArielCalisaya committed:
docs(blog): update blog page hero content

ArielCalisaya committed:
feat(staff): update staff images and linkedin url

Update staff member images from jpg to webp format and correct Laura's LinkedIn URL

ArielCalisaya committed:
feat(press): enhance press page with awards section and marquee animation

- Add awards recognition section with vertical marquee animation
- Update press page content and styling
- Extend marquee component to support vertical animation and custom styling
- Modify TwoColumn component to support multiple buttons and footer text

ArielCalisaya committed:
style(components): improve code formatting and readability

- Fix indentation in TwoColumn component
- Break long props into multiple lines for better readability in OurPartners and Marquee components
- Remove console.log statement in press template
- Consistently format styled-components props and object literals

ArielCalisaya committed:
feat: update students and projects page content and styling

- Adjust margin spacing in the template for better visual hierarchy
- Update page titles and descriptions to better reflect bootcamp outcomes
- Improve SEO titles and content in both English and Spanish versions

ArielCalisaya committed:
fix: remove optional chaining from slug replacement

The optional chaining operator was unnecessary since frontmatter.slug is always expected to be present

ArielCalisaya committed:
docs(press): update press page heading to plural form

ArielCalisaya committed:
feat(utils): add flattenFormData utility for form processing

Introduce a reusable utility function to flatten form data objects, handling both nested values and exclusions. Replace manual form data processing in multiple action functions with this utility to improve consistency and maintainability.

ArielCalisaya committed:
style(utils): format code with consistent string quotes and line breaks

ArielCalisaya committed:
docs: update students and projects page header text

Update punctuation in header paragraph for better readability
